cmake/linux/toolchain-s390x.cmake

```diff
@@ -0,0 +1,25 @@
+set (CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
```

Comment:
Please add an `include_guard` - see the other toolchain files in "cmake/linux/" for examples.

Reply:
Updated as suggested